packages/data-models/appConfig.ts

```diff
@@ -83,13 +83,15 @@ const APP_ROUTE_DEFAULTS = {
   ],
 };
 
-export type IHeaderColourOptions = "primary" | "secondary" | "none";
+export type IHeaderFooterBackgroundOptions = "primary" | "secondary" | "none";
 export type IHeaderVariantOptions = "default" | "compact";
 
 const APP_HEADER_DEFAULTS = {
-  title: "App",
   collapse: false,
-  colour: "primary" as IHeaderColourOptions,
+  colour: "primary" as IHeaderFooterBackgroundOptions,
+  show: true,
```

Comment:
nit(blocking): would make more sense to set `hide: true` as a way of hiding the header given the default value of `show` is true